rocAL/rocAL/source/loaders/image/image_loader.cpp

```diff
@@ -254,6 +254,12 @@ bool ImageLoader::is_out_of_data()
 {
     return (remaining_count() < _batch_size);
 }
+
+size_t ImageLoader::last_batch_padded_size()
+{
```

Comment:
Change braces to google style
Applicable to all such instances in this PR

Reply:
This file does not follow google coding standards. so adding the APIs as the format currently present in this file